x/auction/types/auctions.go

```diff
@@ -66,15 +68,35 @@ func (a BaseAuction) GetType() string { return "base" }
 
 // Validate verifies that the auction end time is before max end time
 func (a BaseAuction) Validate() error {
-	if a.EndTime.After(a.MaxEndTime) {
-		return fmt.Errorf("MaxEndTime < EndTime (%s < %s)", a.MaxEndTime, a.EndTime)
+	// ID can be 0 for surplus, Debt and collateral auctions
+	if strings.TrimSpace(a.Initiator) == "" {
+		return errors.New("auction initiator cannot be blank")
 	}
 	if !a.Lot.IsValid() {
 		return fmt.Errorf("invalid lot: %s", a.Lot)
 	}
+	// NOTE: bidder can be nil for Surplus and Collateral auctions
+	if a.Bidder != nil && a.Bidder.Empty() {
```

Comment:
This check is failing on 
```
go test ./app -run TestAppImportExport -Enabled -Commit -NumBlocks=10 -BlockSize=200 -Seed 4 -v -timeout 24h
```

pretty sure it needs to be 
```suggestion
	if !a.Bidder.Equals(sdk.AccAddress{}) && a.Bidder.Empty() {
```